### PR TITLE
Add template readmes for merged objects 

### LINKED ISSUE #582: Create README templates for merged downloads
_Please label your issue with a label to specify its priority: breaking, high, medium, or low._

### Context

In preparation for downloading merged objects on the Portal, we will need to include README files with each download. The contents of the merged downloads will be different than the existing sample structure, so we can't use the existing templates. 

### Solution or next step

Create two new README templates, one for merged SCE downloads and one for merged AnnData downloads. 
Just noting we might need to stack this on `ark/549-add-merged-objects-support`.

### PR DESCRIPTION
**Pull request by allyhawkins**
## Issue Number

Closes #582 

## Purpose/Implementation Notes

Here I'm adding two new template readmes, one for merged SingleCellExperiment objects and one for merged AnnData objects. The contents of which mostly mirror what's in the other readmes. The main difference here is in the `contents` section where I describe what files are included in the download. 

Just a note that I'm stacking this on `ark/549-add-merged-objects-support` as mentioned in #582. That branch does have an out of date readme, since the cell type report is not mentioned, but I assume it will eventually get merged in with where the cell type changes are, so I didn't fix any of the existing readmes to avoid any future conflicts. 
Also let me know if I should stack this on a different branch. 

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by davidsmejia — COMMENTED** [review]
LGTM - will resolve the out of date readme when we get ready to put this on staging

**Review by davidsmejia — APPROVED** [review]
forgot to approve

File: api/scpca_portal/templates/readme_merged.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+The date you accessed and downloaded this data is included in the name of the top-most directory of your download for your convenience.
+Use this date to populate the access date below.
+
+#### APA Format
+
+Alex's Lemonade Stand Foundation Childhood Cancer Data Lab. (n.d.). The Single-cell Pediatric Cancer Atlas Portal. Retrieved (insert access date), from https://scpca.alexslemonade.org/
+
+#### MLA Format
+
+The Single-Cell Pediatric Cancer Atlas Portal, Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab, https://scpca.alexslemonade.org/. Accessed (insert access date).
```
Comment (jashapiro) [inline api/scpca_portal/templates/readme_merged.md:68]: I don't know if this change affects all readmes?
```suggestion
The Single-cell Pediatric Cancer Atlas Portal, Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab, https://scpca.alexslemonade.org/. Accessed (insert access date).
```

**Review by jashapiro — APPROVED** [review]
LGTM!

I just had one minor comment on capitalization in the citation title

File: api/scpca_portal/templates/readme_merged.md (hunk 1 above)
Comment (allyhawkins) [reply api/scpca_portal/templates/readme_merged.md:68]: It does, so I'll make that change in all of them here.

**Pull request merged**